webapp/packages/core-links/src/GithubLinks.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+export class GithubLinks {
```

Comment:
Seems like its all static, we could convert it to const and function

Reply:
done